### PR TITLE
create animals table

### PR DESCRIPTION
**Pull request by naji-97**
- [ ] Create a table named animals
- [ ] Copy the SQL I used in the previous point into the schema.sql file in the generated repository.
- [ ] Copy the SQL I used in the previous point into the data.sql file in the generated repository.
- [ ] Take a screenshot of the results of my queries.
- [ ]  screenshot 
- [ ]  [query1.](https://github.com/naji-97/Vet-Clinic-dataBases/blob/feature/animal-table/img/vet-clinic-1.png)
- [ ]  [query2](https://github.com/naji-97/Vet-Clinic-dataBases/blob/feature/animal-table/img/vet-clinic-2.png)

### CODE REVIEW

**Review by 1GORDON — COMMENTED** [review]
##
## Hi @naji-97 ✋

Good job so far!

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

### This project will be marked as Invalid and won't affect the number of your weekly reviews.

- [ ] Kindly customize your readme file properly. You may fix the broken issues link to make it more professional.
- [ ] You may include 1-3 descriptive potential features to be added. Add this to your Future Features section.

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql [hunk 1]
```diff
@@ -1,5 +1,12 @@
...
+-- Create a table named animals
 CREATE TABLE animals (
-    name varchar(100)
+  id SERIAL PRIMARY KEY,
+  name VARCHAR(255),
+  date_of_birth DATE,
+  escape_attempts INTEGER,
+  neutered BOOLEAN,
+  weight_kg DECIMAL
 );
```
Comment (DeliceLydia) [inline schema.sql:12]: - [ ] Please add a new column called `species` in the `animal` table

File: data.sql [hunk 2]
```diff
@@ -3,3 +3,11 @@
...
 INSERT INTO animals (name) VALUES ('Daisy');
 INSERT INTO animals (name) VALUES ('Charlie');
+
+-- create table
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES
+  ('Agumon', '2020-02-03', 0, true, 10.23),
+  ('Gabumon', '2018-11-15', 2, true, 8),
+  ('Pikachu', '2021-01-07', 1, false, 15.04),
+  ('Devimon', '2017-05-12', 5, true, 11);
```
Comment (DeliceLydia) [inline data.sql:13]: - [ ] Please add all data provided in the `animal` table apparently you did not add
any

File: queries.sql [hunk 3]
```diff
@@ -1,3 +1,30 @@
...
+
+-- Find all animals that are neutered:
+SELECT * FROM animals WHERE neutered = true;
+
+-- Find all animals not named Gabumon:
+SELECT * FROM animals WHERE name != 'Gabumon';
+
+-- Find all animals with a weight between 10.4kg and 17.3kg (including animals with weights equal  10.4kg or 17.3kg):
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
```
Comment (DeliceLydia) [inline queries.sql:30]: - [ ] Please create queries to answer questions provided

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @naji-97, 

Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [ ] Good README
- [ ] Followed git workflow

## Required Changes ♻️
_Check the comments under the review._



### Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mukhammadrizooff — APPROVED** [review]
Hi @naji-97, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Correct GitFlow ✔️ 
- Good implementations ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**